charts/spinnaker/templates/poddisruptionbudget/pdb.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.pdb.create }}
+{{- range .Values.pdb.services }}
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "spinnaker.fullname" $ }}-{{ . }}
+  labels:
+{{ include "spinnaker.standard-labels" $ | indent 4 }}
+spec:
+  {{- if $.Values.pdb.minAvailable }}
+  minAvailable: {{ $.Values.pdb.minAvailable }}
+  {{- end }}
+  {{- if $.Values.pdb.maxUnavailable }}
+  maxUnavailable: {{ $.Values.pdb.maxUnavailable }}
+  {{- end }}
+  selector:
+    matchLabels: 
+      app.kubernetes.io/name: {{ . }}
+      app.kubernetes.io/part-of: spinnaker
```

Comment:
Ideally, we match on a (release/component) pair, in order to allow for multiple instances of Spinnaker to run simultaneously.

I here did not do so as:
1. We do not have a release label (which are controlled by Halyard). I could be added in our spinnaker config, but it felt unecesary.
```   
app: spin
app.kubernetes.io/managed-by: halyard
app.kubernetes.io/name: gate
app.kubernetes.io/part-of: spinnaker
app.kubernetes.io/version: 1.31.2
cluster: spin-gate
```
2. We'll likely not need in the future 2 spinnaker beasts in the same namespace.